inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -448,3 +561,450 @@ export function createSubscribable<T>(signal: () => T): Subscribable<T> {
 		},
 	})
 }
+
+// --- serialization of loading / saving messages.
```

Comment:
1. move the code to `storage/locking` to declutter `loadProject`? 

2. @martin-lysk @jldec are you sure that the variables should not be scoped within `loadProject`?

Reply:
Will cleanup this file in separate PR - it's still helping us to have these together while we debug locking logic.

Reply:
@samuelstroschein I moved the message load and save state up into loadProject scope
@martin-lysk offered to review
https://github.com/opral/monorepo/pull/2108/commits/b066babde301e54277c9e3ee7ab2d501a5cbf4ed